qipr_approver/approver/views/shib.py

```diff
@@ -0,0 +1,23 @@
+from django.shortcuts import render
+from django.http import HttpResponse, Http404
+from django.views.decorators.csrf import csrf_protect
+
+from approver.workflows import shib
+from approver.constants import SHIB_ENABLED
+
+#TODO: implement real shib and get rid of this fake shib endpoint
+@csrf_protect
```

Comment:
Yes, I think this is still important for when the DJANGO auth system uses post to log in. Pretty much will only happen if there is no Shibboleth.